packages/core/src/components/TermList/Term.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import Row from '../Row';
+import Spacing from '../Spacing';
+import Text from '../Text';
+
+export type Props = {
+  /** Term label describing the value displayed. */
+  label: string;
+  /** Detail links to go after Term label. */
+  after?: React.ReactNode;
+  /** If enabled, detail links at aligned to the right end. */
+  endAlign?: boolean;
+  /** If enabled, term label is regular size instead of small. */
+  regular?: boolean;
+  children: NonNullable<React.ReactNode>;
+};
+
+export default function Term({ label, after, endAlign, regular, children }: Props) {
+  return (
+    <>
+      <dt>
+        <Row after={endAlign && after}>
+          <Text inline small={!regular}>
```

Comment:
I have some designs that uppercases everything. May I get an `uppercased` prop here? :P

Reply:
I think this is usually worse for readability, but added